internal/core/properties.rs

```diff
@@ -456,6 +456,19 @@ struct PropertyHandle {
     handle: Cell<usize>,
 }
 
+impl core::fmt::Debug for PropertyHandle {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        let handle = self.handle.get();
+        write!(
+            f,
+            "PropertyHandle {{ handle: {}, is_locked: {}, is_list: {} }}",
```

Comment:
```suggestion
            "PropertyHandle {{ handle: 0x{:x}, locked: {}, binding: {} }}",
```

I think it makes sense to use hexadecimal for the handle since it is a pointer
you got the "list" reverse, and think it is better to say that it is a binding.
(when not a binding, it is a list of dependent)